packages/jest-cli/src/watch.js

```diff
@@ -225,6 +228,9 @@ const watch = (
           {header: activeFilters(argv)},
         );
         break;
+      case KEYS.R:
+        openReportInBrowser(initialGlobalConfig.coverageDirectory);
```

Comment:
If this returns false, we should probably print an error to the console saying that the browser cannot be opened + the error that is thrown.